test/src/kbasesearchengine/test/integration/SearchAPIIntegrationTest.java

```diff
@@ -549,6 +560,125 @@ public void narrativeDecoration() throws Exception {
         NarrativeInfoDecoratorTest.compare(res.getAccessGroupNarrativeInfo(), expected);
     }
 
+    public void optionalAuthForSearch() throws Exception {
```

Comment:
@MrCreosote so this was not added as a test which may be why the coverage was so low...